packages/cbioportal-utils/src/exon/ExonUtils.ts

```diff
@@ -61,13 +61,63 @@ export function extractExonInformation(
     return exonInfo;
 }
 
-export function formatExonLocation(exonLocation: number) {
+export function formatExonLocation(exonLocation: number, index?: number) {
```

Comment:
Should we add more comments for this function to explain what we are doing and why we are doing it? Also maybe add some tests to cover each case in this function?